### ISSUE TITLE
[BUG] Webpage is showing incorrect package information

### ISSUE THREAD

**Issue by csvn** [npm/cli#557]
# What / Why

> The NPM website is showing incorrect information for the `redux-observable` package.
 CLI commands `npm install` and `npm info` shows the following:
```
$ npm i redux-observable@next
+ redux-observable@2.0.0-alpha.0
...
```
```
$ npm info redux-observable
redux-observable@1.2.0 | MIT | deps: none | versions: 37
...
dist-tags:
latest: 1.2.0        next: 2.0.0-alpha.0
``` 
> The website says there are 36 versions, and `next: 1.0.0-beta.1`
> 
> I apologize if this issue is not in the correct space, but I could not find a better place for this issue since https://npm.community/ is now read-only, and most links still point to this issue tracker.

## When

* The incorrect info has been shown since last publish, ~21 days ago

## Where

* https://www.npmjs.com/package/redux-observable?activeTab=versions

### Current Behavior

* The website is missing version `2.0.0-alpha.0` for `redux-observable`

### Steps to Reproduce

1. Go to https://www.npmjs.com/package/redux-observable?activeTab=versions
2. Compare with output from `npm info redux-observable`
3. The number of versions and the `dist-tags` do not match

### Expected Behavior

* The website should display all versions for `redux-observable` 

## Who

* n/a

## References

* n/a

**Comment by ljharb** [comment]
That’s because it’s tagged as “next”, not “latest”. Nothing seems incorrect here to me.

**Comment by csvn** [comment]
@ljharb I don't understand, are you saying that `next` on the webpage and `next` in the NPM CLI client should not be the same thing?

![image](https://user-images.githubusercontent.com/8770194/70251246-6de17900-177f-11ea-9543-9c716b0e6a6e.png)
![image](https://user-images.githubusercontent.com/8770194/70251333-97020980-177f-11ea-9469-bd723e779240.png)

**Comment by ljharb** [comment]
Ah, my mistake - i agree those should be the same, and 21 days since the publish of “next” is beyond a cdn/caching issue.

**Comment by csvn** [comment]
No problem 🙂

**Comment by esetnik** [comment]
Relates to #545

**Comment by damian-pastorini** [comment]
Hi everyone, I'm having a similar issue, published my package today:
```
npm notice total files:   130
npm notice
+ reldens@4.0.0-beta.1
```
But still hours later I can't still see the new version neither install it or obviously can't publish it again:
`npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/reldens - You cannot publish over the previously published versions: 4.0.0-beta.1.`

Doing some research about this, I found this https://github.com/npm/registry that says I should post on the community but the community is actually closed, so I wasn't sure if create a new issue here was the way to go (though this comment asking about it on this already open).

The weird part is that I can't even see the update in the registry: https://registry.npmjs.org/reldens

Is this something for I should keep waiting until the cache gets invalidated? Or where can I report it?

**Comment by darcyclarke** [comment]
npm `v6` is no longer in active development; We will continue to push security releases to `v6` at our team's discretion as-per our [Support Policy](https://github.com/npm/cli/wiki/Support-Policy).

If your bug is preproducible on `v7`, please re-file this issue using our [new issue template](https://github.com/npm/cli/issues/new?assignees=&labels=Bug%2CNeeds+Triage%2CRelease+7.x&template=bug_7.yml&title=%5BBUG%5D+%3Ctitle%3E).

If your issue was a feature request, please consider opening a new [RRFC](https://github.com/npm/rfcs/issues) or [RFC](https://github.com/npm/rfcs). If your issue was a question or other idea that was not CLI-specific, consider opening a discussion on our [feedback repo](https://github.com/npm/feedback)

**Closing:** This is an automated message.
